dev_guide/projected_volumes.adoc

```diff
@@ -145,26 +145,82 @@ spec:
       - secret:
           name: pass
 ----
++
+The `user` and `pass` values are made up and can be anything valid (base64 encoded).
 
 . Create files containing the secrets:
 +
+For example:
++
 ----
-echo -n "admin" > ./username.txt
-echo -n "1f2d1e2e67df" > ./password.txt
+nano secret.yaml
+----
++
+Enter the following, replacing the password and user information as appopriate:
 ----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mysecret
+type: Opaque
+data:
+  pass: MWYyZDFlMmU2N2Rm
+  user: YWRtaW4=
+---
 
 . Use the following command to create the secrets:
 +
 ----
 oc create -f <filename>
 ----
++
+For example:
++
+----
+oc create -f secret.yaml
+secret "mysecret" created
+----
 
 . Create the pod that references the secret:
 +
 ----
 oc create -f <your_yaml_file>.yaml
 ----
++
+For example:
++
+----
+oc create -f secret-pod.yaml
+pod "test-projected-volume" created
+----
 
+. Verify that the Pod's Container is running, and then watch for changes to
+the Pod:
+
+----
+oc get pod test-projected-volume
+----
+
+The output looks like this:
+
+----
+NAME                    READY     STATUS    RESTARTS   AGE
+test-projected-volume   1/1       Running   0          14s
+
+. In another terminal, get a shell to the running Container:
+
+---
+oc exec -it test-projected-volume -- /bin/sh
+----
+
+. In your shell, verify that the `projected-volumes` directory contains your projected sources:
+
```

Comment:
Unfortunately after looking closer, the projections is wrong. It should be like this for the pod spec:

```
apiVersion: v1
kind: Pod
metadata:
  name: test-projected-volume
spec:
  containers:
  - name: test-projected-volume
    image: busybox
    args:
    - sleep
    - "86400"
    volumeMounts:
    - name: all-in-one
      mountPath: "/projected-volume"
      readOnly: true
  volumes:
  - name: all-in-one
    projected:
      sources:
      - secret:
          name: mysecret
```

And then you can check in the container to see:
```
/ # ls projected-volume/
pass  user
```